extra-x11/picom/spec

```diff
@@ -0,0 +1,3 @@
+VER=8.2
+SRCTBL="https://github.com/yshui/picom/archive/v8.2.tar.gz"
```

Comment:
Please switch to our new `spec` format, as specified in our [ACBS Documentation](https://wiki.aosc.io/developer/packaging/acbs/spec-format/).